tasks/managed/extract-index-image/tests/test-extract-index-image.yaml

```diff
@@ -61,6 +61,28 @@ spec:
                 }
               }
               EOF
+              cat > "$(workspaces.data.path)/internal-request-results.json" << EOF
+              {
+                "index_image": {
+                  "target_index": "quay.io/scoheb/fbc-target-index-testing:v4.12"
+                },
+                "components": [
```

Comment:
We are adding support for multiple components, so this test should probably have several components, no?

Reply:
added.